fiatconnect-api.md

```diff
@@ -762,7 +764,7 @@ of fiat currency the user should expect to recieve in exchange for `cryptoAmount
 The `quote.fiatType`, `quote.cryptoType`, `quote.fiatAmount`, and `quote.cryptoAmount` fields in the response body MUST correspond to the query parameters provided to the endpoint.
 The `quote.guaranteedUntil` field represents the time that the quote is guaranteed until, as a UNIX timestamp. A server MAY choose to provide this value. If it does not, the client should assume that
 the server makes no guarantees about the actual conversion rate when performing a transfer for the given quote. If a server provides this value in the response, it MUST honor the
-provided conversion rate when initiating transfers until the time indiciated in the `quote.guaranteedUntil` field.
+provided conversion rate when initiating transfers until the time indicated in the `quote.guaranteedUntil` field.
```

Comment:
(I know this was just a typo fix, but leaving this here anyways.) Even though we don't actually have any way to verify this, it may also make sense to require servers to return a `quoteId` for transfers out. While we have no way of knowing if servers are actually honoring conversion rates for these transfers, it may result in some providers actually enforcing guarantees for transfers out, and it makes the responses for quotes in and out uniform, which is a plus.

Reply:
Actually, looking further down, it looks like you updated the spec for both `/transfer/in` and `/transfer/out`, so somewhat confused what the intended behavior is.

Reply:
I mistakenly left out transfers out, meant to include both